ds3-sdk/build.gradle

```diff
@@ -83,5 +83,7 @@ dependencies {
     compile     'com.fasterxml.jackson.dataformat:jackson-dataformat-xml:2.7.1'
     compile     'com.google.guava:guava:20.0'
     compile     'com.google.code.findbugs:annotations:3.0.1'
+    compile      group: 'org.apache.commons', name: 'commons-lang3', version: '3.6'
```

Comment:
It seems like we could write our own Pair class rather than adding a new dependency.